functions/system.sh

```diff
@@ -217,6 +220,19 @@ pine64_platform_scripts() {
   fi
 }
 
+pine64_gpio_permissions() {
+  cond_redirect groupadd gpio
+  cond_redirect cp $BASEDIR/includes/PINE64-80-gpio-noroot.rules /etc/udev/rules.d/80-gpio-noroot.rules
+  cond_redirect sed -i -e '$i \chown -R root:gpio /sys/class/gpio \n' /etc/rc.local
+  cond_redirect sed -i -e '$i \chmod -R ug+rw /sys/class/gpio \n' /etc/rc.local
```

Comment:
why so complicated?

Reply:
As I understand it from [this post](https://forum.pine64.org/showthread.php?tid=2233) permission is lost every reboot due to a virtual filesystem. They need to be reapplied every boot, correct me if I am wrong. This works for me at least.

Reply:
Sorry, I didn't express myself clearly. I was confused about the fairly odd sed line but now I understood their meaning. 
Okay then, seems like a good solution ;)